lending_pool/src/utils.rs

```diff
@@ -0,0 +1,427 @@
+elrond_wasm::imports!();
```

Comment:
This file should be cleaned up, there is much commented code, and 427 lines of code is a bit much for an "utils" file, so some refactoring into multiple files is also probably needed.